fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -557,6 +570,7 @@ public void write(DataOutput out) throws IOException {
 
         out.writeLong(replicaQuotaSize);
         dbProperties.write(out);
+        out.writeLong(transactionQuotaSize);
```

Comment:
We can use dbProperties to save this, so that we don't need to upgrade meta version

Reply:
> We can use dbProperties to save this, so that we don't need to upgrade meta version

OK. let me change it